app/Post_Types/Archive_Item/Post_Meta.php

```diff
@@ -50,37 +73,108 @@ public function add_post_meta() {
 			->add_fields( array(
 				$this->get_date_field(),
 			))
-			->set_priority( 'high' );
+			->set_priority( 'default' );
 
 
 		$this->container = Container::make( 'post_meta', 'Related Items' )
 			->where( 'post_type', '=', Archive_Item::NAME )
 			->add_fields( array(
 				$this->get_field_related(),
 			))
-			->set_priority( 'high' );
+			->set_priority( 'default' );
 
 		$this->container = Container::make( 'post_meta', 'Audio' )
 			->where( 'post_type', '=', Archive_Item::NAME )
 			->add_fields( array(
 				$this->get_field_audio(),
 				$this->get_field_audio_oembed()
 			))
-			->set_priority( 'high' );
+			->set_priority( 'default' );
 
 		$this->container = Container::make( 'post_meta', 'Video' )
 			->where( 'post_type', '=', Archive_Item::NAME )
 			->add_fields( array(
 				$this->get_field_video_oembed()
 			))
-			->set_priority( 'high' );
+			->set_priority( 'default' );
 
 		$this->container = Container::make( 'post_meta', 'Document' )
 			->where( 'post_type', '=', Archive_Item::NAME )
 			->add_fields( array(
 				$this->get_field_document()
 			))
-			->set_priority( 'high' );
+			->set_priority( 'default' );
+
+	}
+
+	/**
+	 * Adds one or more classes to the body tag in the dashboard.
+	 *
+	 * @param  \WP_Post $post Current field post.
+	 * @return String          Class name.
+	 */
+	public function get_class( $post ) {
+		$type = carbon_get_the_post_meta( FieldPostMeta::FIELD_TYPE, 'carbon_fields_container_field_variables' );
+		$name = null;
+		switch ($type) {
+			case Text_Field::NAME:
+				$name = Text_Field::class;
+				break;
+			case Date_Field::NAME:
+				$name = Date_Field::class;
+				break;
+			case CPT_Field::NAME:
+				$name = CPT_Field::class;
+				break;
+		}
+		return $name;
+	}
+
+	public function get_field($type) {
+		$id = carbon_get_the_post_meta( FieldPostMeta::FIELD_ID );
+		$label = carbon_get_the_post_meta( FieldPostMeta::FIELD_LABEL_TITLE );
+		$helper = carbon_get_the_post_meta( FieldPostMeta::FIELD_ADMIN_HELPER_TEXT);
+		$static_function_name = sprintf(
```

Comment:
This is convenient but any IDE, even smart ones, will not find it as a usage leading to code unreadability.
Since we're not talking about hundreds of field types I'd go with a map here:
```php
$map = [
	'foo' => Foo_Field::class,
	'bar' => Bar_Field::class,
];

if(!array_key_exists($type, $map)){
	// developer-land exception, let's make it clear
	throw \Diviner\CarbonFields\Errors\UndefinedType("{$type} is not a valid field type");
}

return call_user_func([$map[$key], 'render']);
```

There are many enhancements that could be done here like avoiding static calls, at all, anywhere.
Still this would be a good start.

To avoid instances where the `render` method might not be defined on the field class let's use an interface each field class will implement.

Reply:
good thought. done